sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/dynamodb/DynamoDBIOTest.java

```diff
@@ -51,6 +55,8 @@
 import org.junit.rules.ExpectedException;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
+import org.slf4j.helpers.MessageFormatter;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
```

Comment:
Was it intentional to use the testcontainers shaded guava here?

Reply:
No, an oversight indeed! thanks für catching 👍